pkg/cmd/plugincheck2/quickmeta.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+// GetIDAndVersion Gets plugin id and version from extracted zip
+func GetIDAndVersion(archiveDir string) (*string, *string, error) {
+	fis, _ := ioutil.ReadDir(archiveDir)
```

Comment:
Are we certain the archive directory exists here? If it doesn't, it'll panic on the next line.

Reply:
good catch will update

Reply:
redid this section to handle more edge cases